### DISCUSSION TITLE
Duda sobre práctica Vue

### DISCUSSION THREAD

**Discussion by JavierErazo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#531] [Q&A]
Buenas noches a todos.

Mi duda es acerca de los datos ingresados en el formulario. Uno debe ser capaz de llenarlo y enviarlo, de tal manera que los datos ingresados se reflejen en el componente de pedidos del pastelero, ¿correcto?. ¿Estos datos deben sobrevivir si se actualiza y/o cierra el browser o no es necesario?

Muchas gracias de antemano.

Un saludo

**Comment by doguedogue** [comment]
Asi es, tienes toda la razón. 

Deben viajar de pantalla a pantalla (state), creo que no sobreviven si se cierra, me parece que es a nivel sesión.

**Reply by AngelCruzO** [reply]
Correcto, se mueren una vez se cierra el sitio. Pero mientras estas navegando, se mantienen sin problema

**Reply by MarcosSilvaG** [reply]
podria intentarse con localstorage para que persistan los datos siempre y cuando no cambie la sesión :D

**Comment by Dfredude** [comment]
Solo se mantienen en el state de la sesion, para tener algo que sea permanente se necesita un back-end, el cual no es necesario en base a los requerimientos de la practica. Saludos!
